adhocracy4/ratings/static/ratings/react_ratings.jsx

```diff
@@ -23,8 +25,10 @@ var RatingBox = React.createClass({
   },
   handleRatingModify: function (number, id) {
     api.rating.change({
-      object_pk: this.props.objectId,
-      content_type: this.props.contentType,
+      urlReplaces: {
```

Comment:
I think both options have value. @vellip's approach is more explicit, the old approach had nice separation of abstraction layers. For me, both is fine.